common/changes/@uifabric/experiments/miwhea-card-styles_2019-02-15-23-35.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Update Card styles and add tokens",
+      "type": "patch"
```

Comment:
Should be minor as you are adding new props.

Reply:
Good catch, updated!